pick.html

```diff
@@ -1433,8 +1433,8 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
       </ol>  
 
       <figure>
-          <img style="width:80%", src="figures/exercises/manipulability.png"/>
-          <figcaption>Manipulability ellipsoids for two different postures of the planar two-link manipulator. Source: <elib>Lynch17</elib>.</figcaption>
+          <img style="width:80%", src="figures/exercises/manipulability_edit.png"/>
```

Comment:
Please just replace the existing png?  git remembers.  We don't need both copies sticking around.

Reply:
Good point, fixed